training/data/training.material.csv

```diff
@@ -0,0 +1,6 @@
+id,name,type_id:id
+material_pen,Pen,material_type_physical
+material_study_book,Study book,material_type_physical
+material_notebook,Notebook,material_type_physical
+material_username,User name,material_type_electronic
+material_password,Password,material_type_electronic
```

Comment:
I [found](https://github.com/odoo/odoo/blob/52484b0132ee3e0063dd67cc3c2eb434f99f5db8/openerp/addons/base/module/module_data.xml#L76) the correct name: use "Events"